AnnoyancesFilter/sections/tweaks.txt

```diff
@@ -1311,8 +1311,7 @@ my-expert.ru#$#html { -webkit-touch-callout: default!important; -moz-user-select
 codedosa.com,my-expert.ru#$#.unselectable { -webkit-touch-callout: default!important; -moz-user-select: text!important; -webkit-user-select: text!important; -ms-user-select: text!important; -o-user-select: text!important; user-select: text!important; -khtml-user-select: text!important; }
 fmkorea.com#$##bd_capture { background: none!important; }
 page.kakao.com#$#div[id="epubViewerBody"][class^="css-"] { -webkit-touch-callout: default!important; -moz-user-select: auto!important; -webkit-user-select: auto!important; -ms-user-select: auto!important; -o-user-select: auto!important; user-select: auto!important; -khtml-user-select: auto!important; }
-webtoon.kakao.com,page.kakao.com#$#body { -webkit-touch-callout: default!important; -moz-user-select: auto!important; -webkit-user-select: auto!important; -ms-user-select: auto!important; -o-user-select: auto!important; user-select: auto!important; -khtml-user-select: auto!important; }
-cafe.daum.net#$#body * { -webkit-touch-callout: default!important; -moz-user-select: auto!important; -webkit-user-select: auto!important; -ms-user-select: auto!important; -o-user-select: auto!important; user-select: auto!important; -khtml-user-select: auto!important; }
+cafe.daum.net,webtoon.kakao.com,page.kakao.com#$#body { -webkit-touch-callout: default!important; -moz-user-select: auto!important; -webkit-user-select: auto!important; -ms-user-select: auto!important; -o-user-select: auto!important; user-select: auto!important; -khtml-user-select: auto!important; }
```

Comment:
What's this rule for? It seems old rules are still needed. Btw, `List-KR` have similar rule.

Reply:
The text of `cafe.naver.com` cannot be copied on Firefox.

Reply:
Ok, I got it. Please additionally revert `cafe.daum.net#$#body * { -webkit-touch-callout: default!important; -moz-user-select: auto!important; -webkit-user-select: auto!important; -ms-user-select: auto!important; -o-user-select: auto!important; user-select: auto!important; -khtml-user-select: auto!important; }`.

Reply:
I think we don't need to revert before https://github.com/AdguardTeam/AdguardFilters/pull/90884/commits/6cbd0eec2981322088c49196a347050546d1bf28.
https://user-images.githubusercontent.com/37463155/129352277-7cc0b2ae-6db8-421f-a102-cacbef6cb676.mp4
https://user-images.githubusercontent.com/37463155/129352292-fb5b2be6-f602-465e-ba49-26b991c5d399.mp4

Reply:
I meant that I need two rules for text selection.
```
cafe.daum.net#$#body * { -webkit-touch-callout: default!important; -moz-user-select: auto!important; -webkit-user-select: auto!important; -ms-user-select: auto!important; -o-user-select: auto!important; user-select: auto!important; -khtml-user-select: auto!important; }
cafe.daum.net#$#body { -webkit-touch-callout: default!important; -moz-user-select: auto!important; -webkit-user-select: auto!important; -ms-user-select: auto!important; -o-user-select: auto!important; user-select: auto!important; -khtml-user-select: auto!important; }
```

Reply:
Okay.

Reply:
Done. https://github.com/AdguardTeam/AdguardFilters/pull/90884/commits/b430fd9411809ac7baa78397a8204b5e2aa7980e